src/i_sdlmusic.c

```diff
@@ -645,18 +646,42 @@ static boolean ReadSubstituteConfig(char *filename)
         return false;
     }
 
-//    old_subst_music_len = subst_music_len;
+    length = M_FileLength(fs);
```

Comment:
Please just replace this with a call to `M_ReadFile()`. The only catch is that `M_ReadFile()` will not add the terminating `\0`; it was already suggested in #875 to make it do so. That should simplify things significantly.

Reply:
That PR doesn't mention `M_ReadFile()`, did you mean #872?
Are you sure adding a '\0' doesn't change any other behavior? I've done this but intentionally left the returned `length` unmodified since some of the Strife code looked to use it for copying files.